### PR TITLE
Update 03-supported-databases.mdx

### PR DESCRIPTION
**Pull request by Jolg42**
cross link with system-requirements

With this now both pages links to each other

### CODE REVIEW

File: content/400-reference/300-database-reference/03-supported-databases.mdx [hunk 1]
```diff
@@ -44,3 +44,5 @@ Support for the following databases is available as a Preview:
 | MongoDB     | 4.4+    |
 | PlanetScale | \*      |
 | CockroachDB | 21.2.4+ |
+
+See also: [System requirements](reference/system-requirements)
```
Comment (janpio) [inline content/400-reference/300-database-reference/03-supported-databases.mdx]:
```suggestion
See also: [System requirements](/reference/system-requirements)
```

Comment (Jolg42) [reply content/400-reference/300-database-reference/03-supported-databases.mdx]: thanks!

Comment (Jolg42) [reply content/400-reference/300-database-reference/03-supported-databases.mdx]: but somehow it's not done like that on the other side

**Review by janpio — APPROVED** [review]

**Comment by tanberry** [comment]
Thank you @Jolg42 !

**Pull request merged**